src/main/java/uk/gov/ch/model/payment/ConfirmationStatementPaymentJson.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.ch.model.payment;
+
+public class ConfirmationStatementPaymentJson {
+
+    private Boolean isPaid;
```

Comment:
are you able to annotate with a @JsonProperty? this will define the api contract and clients can always rely on the json field being the same and not changing.